build.gradle

```diff
@@ -103,40 +103,44 @@ test {
 }
 
 shadowJar {
-    archiveName = 'RepoSense.jar'
-    destinationDir = file("${buildDir}/jar/")
+    archiveFileName = 'RepoSense.jar'
+    destinationDirectory = file("${buildDir}/jar/")
 
     manifest {
         attributes 'Implementation-Version': getRepoSenseVersion()
     }
 }
 
-task installFrontend(type: com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask) {
+tasks.register('installFrontend', com.liferay.gradle.plugins.node.tasks.ExecutePackageManagerTask) {
     workingDir 'frontend/'
     args = ['install']
 }
 
-task buildFrontend(dependsOn: ['installFrontend'], type: com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask) {
+tasks.register('buildFrontend', com.liferay.gradle.plugins.node.tasks.ExecutePackageManagerTask) {
+    dependsOn 'installFrontend'
```

Comment:
Would it be possible to do an assignment like `def installFrontend = tasks.register(...)`? This way all further references to `installFrontend` like in line 120 can reference `installFrontend` without quotations directly. At least this seems possible when I tried it locally.

Reply:
I managed to change some of the assignments